pkg/plugin/util/util.go

```diff
@@ -271,3 +273,11 @@ func HasFileContentWith(path, text string) (bool, error) {
 
 	return strings.Contains(string(contents), text), nil
 }
+
+func Log() *logrus.Logger {
```

Comment:
What do you think about creating a cli/util/util.go file and relocating the log utility there? My rationale is that the plugin utils is designed for utilities common to the plugins. In contrast, the log utility is utilized throughout the entire CLI. Structuring it this way might provide a clearer distinction and organization.

Reply:
Okay i will do the suggested changes 
Thank you